### ISSUE TITLE
Upgrade to New version of bookstack when using docker container?

### ISSUE THREAD

**Issue by RaithZ** [solidnerd/docker-bookstack#85]
I have my bookstack up and running fine but im using 0.16.2 verison of bookstack.  How should I go about upgrading to the latest version of bookstack?  Is it as simple as deploying the new docker container with the new version and linking it with the mariadb for bookstack or... what?

**Comment by BeckerFelix** [comment]
However, the version of the bookstack release to use is hardcoded in the Dockerfile. You will only get a new version of Bookstack once solidnerd has updated the Dockerfile. 

Is there a reason why the Dockerfile does not automatically pull the latest bookstack release? 
Same question for the apache-stretch.

**Comment by BeckerFelix** [comment]
I see, thanks for the explanation :)

**Comment by tylwright** [comment]
@solidnerd - Looks like we can go ahead and close this issue.  Thanks!

**Comment by RaithZ** [comment]
Yes this issue can be closed.  What I didn't realize is that bookstack upgrades the database schema automatically when you start the newer version.  I found this out the hard way by trying to go back to the old version and not being able to.  I had to restore the old database to get the old one to work.

**Comment by ccffvv** [comment]
Hello, All,

I am new to bookstack docker: I am trying to update my bookstack v0.31.6 to a newer version.

I have followed the instruction without seeing any error ( I stopped the old container, delete the old image, pull the new image and create a new container)

docker stop <bookstack container name>
docker rm <bookstack container name>
docker pull solidnerd/bookstack

It is all running as before, but it is still showing "BookStack v0.31.6" after the new container. How to update it to a new version like v21.10.1?

Thank you very much.
